npm/parseSelector.go

```diff
@@ -14,18 +14,18 @@ import (
 // An ReqHeap is a min-heap of labelSelectorRequirements.
 type ReqHeap []metav1.LabelSelectorRequirement
 
-func (h ReqHeap) Len() int { 
```

Comment:
Where does NPM use heap?

Reply:
this was earlier used in creating the single podselector for processed NPMap. this is just a lint change, and i do not want to touch this part in this PR, as the scope is different.